buildtest/cli/__init__.py

```diff
@@ -12,6 +13,11 @@
 from buildtest.defaults import console
 from buildtest.schemas.defaults import schema_table
 
+# Variables needed to show all sub commands and their help mesaage
+help1 = "-H" in sys.argv
+help2 = "--help--all" in sys.argv
```

Comment:
can you rename to `--help-all` so no double dash before `all`. Long options are double dash like `--XYZ` and short options are single dash `-X` thats how argparse refers to short vs long options. 
![Screenshot 2023-07-21 at 11 26 20 AM](https://github.com/buildtesters/buildtest/assets/12942230/fb849a58-86f9-4768-a338-a6e8053f2ebe)

Reply:
Just finished shortening it!